src/plugins/credential_management/public/components/breadsrumbs.ts

```diff
@@ -0,0 +1,24 @@
+import { i18n } from '@osd/i18n';
+
+export function getListBreadcrumbs() {
+  return [
+    {
+      text: i18n.translate('credentialManagement.credentials.listBreadcrumb', {
+        defaultMessage: 'Credentials',
+      }),
+      href: `/`,
+    },
+  ];
+}
+
+export function getCreateBreadcrumbs() {
+  return [
+    ...getListBreadcrumbs(),
+    {
+      text: i18n.translate('credentialManagement.credentials.createBreadcrumb', {
+        defaultMessage: 'Create credentials',
+      }),
+      href: `/create`,
+    },
+  ];
+}
```

Comment:
do we plan to have a separate edit page url?

Reply:
Yes, will included in the editing page PR